pnpm-lock.yaml

```
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Huh, I wonder why this was updated since the rest of the lockfile seems the same...

---

Good catch. I ignored that.

PNPM updates the lockfile version when running `pnpm install`. But `6.1` is a value, which we don't want. That's from an old PNPM version and was rolled back. See https://github.com/orgs/pnpm/discussions/6857.

Let's reset that change:

```suggestion
lockfileVersion: '6.0'
```

@Baltazore: Looks as if you need to upgrade PNPM on your machine.

---

yeah I had pnpm 8.6 locked due to ember-data, sorry ...